x/ibc/core/02-client/types/params.go

```diff
@@ -0,0 +1,70 @@
+package types
+
+import (
+	"fmt"
+	"strings"
+
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+)
+
+var (
+	// DefaultAllowedClients are "Solo Machine" and "Tendermint"
+	DefaultAllowedClients = []string{"Solo Machine", "Tendermint"}
```

Comment:
Should `LocalHost` be added as well?

Reply:
we could consider adding it and remove the CreateLocalhost boolean from GenesisState. Although I'd prefer to do that on another PR